packages/styling/src/interfaces/ISemanticTextColors.ts

```diff
@@ -64,17 +64,17 @@ export interface ISemanticTextColors {
   //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
 
   /**
-   * The default color of text indicating error on bodyBackground.
+   * The default color of text on errorBackground.
```

Comment:
No this is wrong, the original is correct.

Reply:
oops, thanks for catching this